src/staking/index.ts

```diff
@@ -443,7 +446,7 @@ export class Staking {
     // Reconstruct and validate the slashingOutputIndex
     const slashingOutputIndex = findMatchingTxOutputIndex(
       slashingTx,
-      deriveSlashingOutputAddress(scripts, this.network),
+      deriveSlashingOutput(scripts, this.network).outputAddress,
```

Comment:
Let's follow the same style in which this is called before as with other methods?